docs/content/reference/resource-schema/cache/redis/index.md

```diff
@@ -76,10 +76,9 @@ The `redislabs.com/Redis` is a [resource]({{< ref portable-resources >}}) which
 
 The following methods are available on the Redis cache:
 
-| Method | Description |
-|--------|-------------|
-| connectionString() | Get the connection string for the Redis cache. |
-| password() | Get the password for the Redis cache. |
+| Method | Description | Example |
+|--------|-------------|---------|
```

Comment:
Should we expand on `.value` in the description, since it might not obvious until someone takes a closer look at the example?